tools/scripts/params/tokenomics_mint_allocation_dao.json

```diff
@@ -0,0 +1,12 @@
+{
+  "body": {
+    "messages": [
+      {
+        "@type": "/poktroll.tokenomics.MsgUpdateParam",
+        "authority": "pokt10d07y265gmmuvt4z0w9aw880jnsr700j8yv32t",
+        "name": "mint_allocation_dao",
+        "as_float": 0.1
```

Comment:
Shouldn't it be `as_double` here? Which I think should be `as_float64`

Reply:
Yes, nice catch! See https://github.com/pokt-network/poktroll/pull/909#discussion_r1837849460 re `double` vs. `float64`.